docs/content/build.md

```diff
@@ -58,3 +58,14 @@ This starts the build process for the worker and deployment type (cloud or local
 > **It’s finished building when:** there is a message stating `Completed build for Local/Cloud target`.
 
 You can now find the built-out workers in `build/assembly/worker` in the root of the SpatialOS GDK for Unity project.
+
+#### TIP: Fast worker build with quick-run
+ When you are developing with the GDK, you can use a quick-run to build workers using Ctrl+L (Windows) or Cmd+L (Mac).
+
+ This tutorial takes you through a cloud deployment. When you are developing with the GDK, you don’t need to always use a cloud deployment; you can use a local deployment instead. In a local deployment, you can either build your workers to run locally or use a quick-run option. With quick-run you can run multiple workers in your Unity Editor, so you don't have to keep building out workers during development iteration. 
```

Comment:
The first sentence sounds slightly weird... how about:
This tutorial takes you through the steps needed to to start up and connect to a cloud deployment.

Also: do we now say "quick-run" option? this sounds like something that needs to be enabled

Reply:
it doesn't really have a name afaik 🤷‍♀️  I'm less against the quick-run part and more against calling it an option as it sounds like it needs to be configured somewhere

Reply:
Made it clearer - and got rid of "option".